### PR TITLE
PEP 561: Add partial stub section

### PR DESCRIPTION
**Pull request by ethanhs**

### CODE REVIEW

File: pep-0561.rst [hunk 1]
```diff
@@ -181,8 +181,23 @@ of that Python version. This can be queried e.g.
...
+
+Partial Stub Packages
+---------------------
+
+Many stub packages will only have part of the type interface for libraries
+completed, especially initially. For the benefit of type checking and code
+editors, packages can be "partial". This means modules not found in the stub
+package SHOULD be searched for in the corresponding runtime package.
+
+Type checkers should merge the stub package and runtime package directories.
```
[the remark this answers is not in the document]
Comment (JelleZijlstra) [reply pep-0561.rst]: Why not? A possible use case could be a stub package that improves on one module within a package that has stubs in typeshed.

[the remark this answers is not in the document]
Comment (ethanhs) [reply pep-0561.rst]: Ah, I misunderstood. I thought you meant other installed stub packages. I don't particularly see this as critical. If you have an issue with typeshed modules, submit a fix. However, I don't particularly see why this interpretation is a bad thing either.

@gvanrossum @vlasovskikh I'd be interested in your thoughts on the issue above. I think I'm leaning toward yes, but I don't think there is a strong argument either way.

File: pep-0561.rst [hunk 2]
```diff
@@ -202,13 +217,16 @@ Acknowledgements
...
-of Ivan Levkivskyi, Jelle Zijlstra, Nick Coghlan, Daniel F Moisset, Nathaniel
-Smith, and Guido van Rossum.
+of Ivan Levkivskyi, Jelle Zijlstra, Nick Coghlan, Daniel F Moisset, Andrey
+Vlasovskikh, Nathaniel Smith, and Guido van Rossum.
 
 
 Version History
 ===============
 
+* 2018-05-015
```
[the remark this answers is not in the document]
Comment (ethanhs) [reply pep-0561.rst]: Fixed, thank you!

**Comment by brettcannon** [comment]
@ethanhs is this ready to be merged?

**Comment by ethanhs** [comment]
@brettcannon Yes it is, thank you!

**Pull request merged**